src/integration-test/java/se/sundsvall/casedata/apptest/util/TestConstants.java

```diff
@@ -2,15 +2,10 @@
 
 public final class TestConstants {
 
-	private TestConstants() {
-	}
+	private TestConstants() {}
```

Comment:
SonarLint plugin wants empty constructors like this to be commented. Like
`// Intentional left empty to prevent instantiation`